test/validate/eia_test.py

```diff
@@ -45,13 +45,13 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_607_752, 1_607_752, 128_252),
+        ("bf_eia923", 1_607_692, 1_607_692, 128_252),
         ("bga_eia860", 142_391, 142_391, 142_391),
-        ("boil_eia860", 83_416, 83_416, 83_416),
-        ("frc_eia923", 650_671, 265_784, 25_443),
+        ("boil_eia860", 83_418, 83_418, 83_418),
+        ("frc_eia923", 656_388, 268_352, 25_443),
         ("gen_eia923", None, 5_179_478, 433_336),
-        ("gens_eia860", 557_368, 557_368, 557_368),
-        ("gf_eia923", 2_918_170, 2_918_170, 246_324),
+        ("gens_eia860", 557_369, 557_369, 557_369),
+        ("gf_eia923", 2_933_836, 2_933_836, 246_324),
```

Comment:
Here we're adding ~15,000 records out of ~3M, so about 0.5% new records, which seems like it's in the right ballpark.